sdk/helper/ldaputil/config.go

```diff
@@ -107,6 +113,16 @@ Default: cn`,
 			Description: "CA certificate to use when verifying LDAP server certificate, must be x509 PEM encoded (optional)",
 		},
 
+		"client_tls_cert": {
+			Type:        framework.TypeString,
+			Description: "Client certificate to provide to the LDAP server, must be x509 PEM encoded (optional)",
```

Comment:
You'll need the `DisplayAttrs` for this field as well

Reply:
Thanks!